### PR TITLE
[4.0] Adding systemtest step for MySQL8

### PR DESCRIPTION
**Pull request by Hackwar**
What the title says.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -240,6 +247,15 @@ services:
       MYSQL_ROOT_PASSWORD: joomla_ut
       MYSQL_DATABASE: test_joomla
 
+  - name: mysql8
+    image: mysql:8
+    command: ["--default-authentication-plugin=mysql_native_password"]
+    environment:
+      MYSQL_USER: joomla_ut
```
Comment (810) [inline .drone.yml:251]: should this not be root?

Comment (Hackwar) [reply .drone.yml:251]: no, the systemtests are using that user for DB access.

**Comment by Hackwar** [comment]
I've pulled in #25600 and signed the drone.yml again to make merging easier.

**Comment by wilsonge** [comment]
I disagree with #25600 for reasons listed there but happy with this one once that is resolved

**Comment by wilsonge** [comment]
Still conflicts anyway :D

**Comment by Hackwar** [comment]
Not anymore 😉

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**